features/cache_control_modification.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Proxy Modification of Cache-Control Header to include max-age when Babbage sets Cache-Control to "public" or "private"
+
```

Comment:
You can add a description here, see how it has been done in the other feature files. You could explain what this does in plain English. For instance:

> There are certain instances in which the Proxy needs to set the max-age directive, but Babbage has already set the Cache-Control header (for instance, to "public" or "private"). In these scenarios, the Proxy has to set the max-age directive without overwriting the existing headers.